pages/all/daos/AllDaosPage.tsx

```diff
@@ -92,57 +89,47 @@ const AllDaosPage: FC<BrowseAllDaosProps> = ({
     [router]
   );
 
-  const handleCreateDao = useCallback(
-    () => (accountId ? router.push(CREATE_DAO_URL) : login()),
-    [login, router, accountId]
-  );
-
   const isLoading = useRouterLoading();
 
   return (
-    <div className={styles.root}>
-      <div className={styles.header}>
-        <h1>{t('allDaos')}</h1>
-        <Button variant="black" size="small" onClick={handleCreateDao}>
-          {t('createNewDao')}
-        </Button>
-      </div>
-
-      <div className={styles.filter}>
-        <Dropdown
-          options={sortOptions}
-          value={activeSort}
-          defaultValue={activeSort}
-          onChange={handleSort}
-        />
-      </div>
-      <InfiniteScroll
-        dataLength={data.length}
-        next={getMoreDaos}
-        hasMore={hasMore}
-        loader={<h4 className={styles.loading}>{t('loading')}...</h4>}
-        style={{ overflow: 'initial' }}
-        endMessage={
-          <p className={styles.loading}>
-            <b>{t('youHaveSeenItAll')}</b>
-          </p>
-        }
-      >
-        <div className={styles.content}>
-          {data.map(item => {
-            return (
-              <DaoDetailsGrid
-                loading={isLoading}
-                key={item.id}
-                dao={item}
-                activeProposals={item.activeProposalCount}
-                totalProposals={item.totalProposalCount}
-              />
-            );
-          })}
+    <DaosList label="allDaos">
+      <>
```

Comment:
I consider you will be able to remove `<></>` when you remove `children` prop description from `DaoList` comopnent.